static_root/js/metadata.js

```diff
@@ -1,3 +1,15 @@
+var project = {
+"cmip5" : "Coupled Model Intercomparison Project 5",
+"cmip6" : "Coupled Model Intercomparison Project 6",
+"cordex" : "Coordinated Regional Climate Downscaling experiment",
+"dyamond_winter" : "DYnamics of the Atmospheric general circulation Modeled On Non-hydrostatic Domains (DYAMOND) Winter Simulations",
+"dcpp" : "Decadal Climate Prediction Project: https://gmd.copernicus.org/articles/9/3751/2016/",
+"miklip-forecast" : "Mittelfristige Klimaprognosen (Medium-term Climate Predictions)",
+"miklip-ini" : "Mittelfristige Klimaprognosen (Medium-term Climate Predictions)",
+"mpi-ge" : "Max Planck Institute for Meteorology (MPI-M) Grand Ensemble (MPI-GE)",
+"nextgems" : "Next Generation Earth Modelling Systems",
+"nukleus" : "Nutzbare lokale Klimainformationen für Deutschland (a cross-sectional activity of RegIKlim)",
+};
```

Comment:
Great, can you add also Dyamond, without winter?

Reply:
sure!

btw, the fs_type is[ here](https://github.com/FREVA-CLINT/freva-web/pull/34/files#diff-62296473567b811e10a1d9617872f01f09eba9aa86f98bb43ae10a4e32f70850R1852)